extensions/data-plane-transfer/data-plane-transfer-core/src/main/java/org/eclipse/dataspaceconnector/transfer/dataplane/core/security/DataEncrypterImpl.java

```diff
@@ -0,0 +1,15 @@
+package org.eclipse.dataspaceconnector.transfer.dataplane.core.security;
+
+import org.eclipse.dataspaceconnector.transfer.dataplane.spi.security.DataEncrypter;
+
+public class DataEncrypterImpl implements DataEncrypter {
```

Comment:
Maybe this could be called `NoopDataEncrypter` and used as default implementation for `DataEncrypter`